### ISSUE TITLE
ARIA for image fields

### ISSUE THREAD

**Issue by BeksOmega** [google/blockly#8255]
Parent Issue: https://github.com/google/blockly/issues/8205

**Comment by BenHenning** [comment]
This should largely be treated the same way as #8254 (that is, incorporated directly within a block label).

**Comment by BenHenning** [comment]
> This should largely be treated the same way as [#8254](https://github.com/google/blockly/issues/8254) (that is, incorporated directly within a block label).

This is incorrect. Sometimes field images can be selected directly (such as when they have a click handler), so some aria work is needed.
